runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentView.java

```diff
@@ -8,14 +8,17 @@
 import android.content.Context;
 import android.os.Build;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.accessibility.AccessibilityNodeProvider;
 import android.view.inputmethod.EditorInfo;
 import android.view.inputmethod.InputConnection;
+import android.view.MotionEvent;
 
 import org.chromium.content.browser.ContentView;
 import org.chromium.content.browser.ContentViewCore;
 
 public class XWalkContentView extends ContentView {
+    private static final String TAG = "XWalkContentView";
```

Comment:
yes, it's not used so far, just put it there for future debug use so that we don't need to add TAG and import android.util.Log back and forth.